### ISSUE TITLE
JAWS reads blank content

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#176]
If JAWS encounters an element with no content it reads 'blank'.  This content should be removed from the DOM.

**Comment by jessegreenberg** [comment]
Closing.
